src/utils/configs.ts

```diff
@@ -10,5 +16,21 @@ export const defaultNMSConfig = () => ({
     http: {
         port: 8087,
         allow_origin: '*'
+    },
+    trans: {
+        ffmpeg,
+        tasks: [{
+            app: 'live',
+            vc: "copy",
+            vcParam: [],
+            ac: "aac",
+            acParam: ['-ab', '64k', '-ac', '1', '-ar', '44100'],
+            rtmp:true,
+            rtmpApp:'live2',
+            hls: true,
+            hlsFlags: '[hls_time=2:hls_list_size=3:hls_flags=delete_segments]',
+            dash: true,
+            dashFlags: '[f=dash:window_size=3:extra_window_size=5]'
+        }]
```

Comment:
For testing/benchmarking purpose, we should separate those streaming protocols into independent instances to avoid extra decoding time.